sematic/types/types/matplotlib/figure.py

```diff
@@ -1,15 +1,20 @@
+# Standard Library
+from io import BytesIO
+
 # Third-party
 import matplotlib.figure  # type: ignore
-from mpld3 import fig_to_dict
 
 # Sematic
 from sematic.types.registry import SummaryOutput, register_to_json_encodable_summary
+from sematic.types.types.image import Image, image_to_summary
 
 
 @register_to_json_encodable_summary(matplotlib.figure.Figure)
```

Comment:
Is it possible to eliminate this type and instead let it directly become `Image` type? So we save the need to have `matplot.tsx` and the type mapping on the front end. 

Can it be a type-to-type conversion with the type key changed as well?

Reply:
Yes, as explained in a comment in `matplot.tsx`:
```
// This component can be entirely removed and replaced with ImageValueView
// a couple of releases after 0.28.0
```
We will not need the `matplot.tsx` anymore after the deprecation window. We can just use `ImageValueView`.

On the Python side, it is still nice to spare users the trouble of having to do this:
```python
file_obj = BytesIO()
figure.savefig(file_obj)
file_obj.flush()
file_obj.seek(0)
image = Image(bytes=file_obj.read())
return image
```
and instead just
```python
return figure
```